tesla_powerwall/responses.py

```diff
@@ -69,6 +69,13 @@ def energy_imported(self):
     def get_energy_imported(self, precision=DEFAULT_KW_ROUND_PERSICION) -> float:
         return convert_to_kw(self.energy_imported, precision)
 
+    @property
+    def instant_total_current(self):
+        return self.assert_attribute("instant_total_current")
+
+    def get_instant_total_current(self, precision=DEFAULT_KW_ROUND_PERSICION) -> float:
+        return round(self.assert_attribute("instant_total_current"), precision)
```

Comment:
Could you replace the `assert_attribute` with the direct access to the property to make it conform with the other methods?